### PR TITLE
executor: support plan replayer capture remove task

### PR DESCRIPTION
**Pull request by Yisaer**
### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tidb/issues/41262

Problem Summary:

### What is changed and how it works?

1. support `plan replayer capture remove "sql_digest" "plan_digest" ` to remove task
2. let plan replayer related table can be visited under sem mode

### Check List

Tests 

- [x] Unit test

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by hawkingrei — APPROVED** [review]

File: executor/plan_replayer.go [hunk 1]
```diff
@@ -102,6 +106,25 @@ func (e *PlanReplayerExec) Next(ctx context.Context, req *chunk.Chunk) error {
...
 }
 
+func (e *PlanReplayerExec) removeCaptureTask(ctx context.Context) error {
+	ctx1 := kv.WithInternalSourceType(ctx, kv.InternalTxnStats)
+	exec := e.ctx.(sqlexec.SQLExecutor)
+	_, err := exec.ExecuteInternal(ctx1, fmt.Sprintf("delete from mysql.plan_replayer_task where sql_digest = '%s' and plan_digest = '%s'",
+		e.CaptureInfo.SQLDigest, e.CaptureInfo.PlanDigest))
+	if err != nil {
+		logutil.BgLogger().Warn("remove mysql.plan_replayer_status record failed",
+			zap.Error(err))
```
Comment (xuyifangreeneyes) [inline executor/plan_replayer.go:116]: Do we also need to log `SQLDigest` and `PlanDigest`?

**Review by xuyifangreeneyes — APPROVED** [review]
rest LGTM

File: executor/plan_replayer.go (hunk 1 above)
Comment (Yisaer) [reply executor/plan_replayer.go:116]: Not necessary I think

**Pull request merged**